docs/products/valkey/howto/migrate-redis-valkey.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Migrate from Redis®* to Aiven for Valkey using Aiven Console
+sidebar_label: Mirgrate from Redis®*
+---
+
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+Migrate your Redis®* databases, whether on-premise or cloud-hosted, to Aiven for Valkey, using Aiven Console's guided wizard.
+
+:::important
+Migrating from Google Cloud Memorystore for Redis is not currently
+supported.
+
+The version of the source Redis service cannot be higher than the version
+of the target Aiven for Caching service.
+:::
+
+## Prerequisites
+
+Before starting the migration process, ensure you have the following:
+
+- A target Aiven for Valkey service. To create one, see
+  [Get started with Aiven for Valkey](/docs/products/caching/get-started).
```

Comment:
the link should point to valkey

Reply:
Thanks for catching these. I thought I had updated all.